external/lmdb/lmdbConfig.cmake.in

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2021 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+include("${CMAKE_CURRENT_LIST_DIR}/lmdbTargets.cmake")
```

Comment:
Afaik this file is generated by the cmake install command, no?

Reply:
lmdbTargets.cmake is generated, but install command requires lmdbConfig.cmake to be present.